src/test/java/com/iexec/worker/compute/pre/PreComputeAppRunnerTests.java

```diff
@@ -56,13 +71,18 @@ void knownCauseTransmitted(EnvironmentVariables environment) throws Exception {
         environment.set(IEXEC_TASK_ID, CHAIN_TASK_ID);
 
         PreComputeApp preComputeApp = mock(PreComputeApp.class);
-        doThrow(new PreComputeException(ReplicateStatusCause.POST_COMPUTE_COMPUTED_FILE_NOT_FOUND))
+        doThrow(new PreComputeException(POST_COMPUTE_COMPUTED_FILE_NOT_FOUND))
                 .when(preComputeApp).run();
 
         WorkerApiClient workerApiClient = mock(WorkerApiClient.class);
 
         when(preComputeAppRunner.createPreComputeApp(CHAIN_TASK_ID)).thenReturn(preComputeApp);
 
+        Field signerServiceField = PreComputeAppRunner.class.getDeclaredField("signerService");
+        signerServiceField.setAccessible(true);
+        signerServiceField.set(preComputeAppRunner, signerService);
```

Comment:
The `org.springframework:spring-test` dependency (to be added in build.gradle in test dependencies) offers much help for this cases.
```suggestion
        ReflectionTestUtils.setField(preComputeAppRunner, "signerService", signerService);
```
This can be added in a `@BeforeEach` annotated method to perform the initialization for each test method.

Reply:
Resolved in [5c69bb7](https://github.com/iExecBlockchainComputing/tee-worker-pre-compute/pull/99/commits/5c69bb7597591e10b29f4130c678519a67fb30ff)